docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: add-return-to-previous-functionality
 title: Add 'Return to previous' functionality
-description: Offer an easy way of return to your context.
+description: Offer an easy way of return to allow users to return to their context.
```

Comment:
@josmperez wdyt of "Allow users to easily return to a specific context"?

Reply:
works for me